vendor/k8s.io/kubernetes/pkg/kubectl/cmd/util/patch.go

```diff
@@ -0,0 +1,31 @@
+package util
+
+import (
+	"fmt"
+	"io"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+)
+
+func PrintSuccess(mapper meta.RESTMapper, shortOutput bool, out io.Writer, resource, name string, dryRun bool, operation string) {
```

Comment:
I see we rely on this function a lot in origin, but there's no reason to introduce a carry here. Why not copy it over to origin?

Reply:
> I see we rely on this function a lot in origin, but there's no reason to introduce a carry here. Why not copy it over to origin?

I'll let @juanvallejo sort it out when OC moves.

Reply:
at least call it `<drop>` and add a comment about when we can drop it